ADRs/0001-adopt-multiaddr.md

```diff
@@ -0,0 +1,60 @@
+# ADR-0001: Adopt Multiaddr
+
+# Adopt Multiaddr for Addressing and Protocol Handling
+
+* Status: in review
+* Deciders: Pocket Network team
+* Date: 2023-04-17
+
+Technical Story: GitHub discussion [#616: Should we go multiaddr](https://github.com/pokt-network/pocket/discussions/616)
+
+## Context and Problem Statement
+
+In the context of simplifying and consolidating node identity, facing the need for a standardized and extensible address format, we decided to adopt multiaddr and neglect alternative address conventions, to achieve a future-proof and composable addressing scheme, accepting the additional dependency and learning curve, because multiaddr provides a standard solution with many benefits.
```

Comment:
Nit: I think the problem statement section shouldn't include what decision was made and should instead focus on specifically the problems. The decision section later in the document should be where the decision is stated and expanded on if necessary.